internal/service/backup/vault_default.go

```diff
@@ -0,0 +1,106 @@
+package backup
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/backup"
+
+	"github.com/hashicorp/aws-sdk-go-base/v2/awsv1shim/v2/tfawserr"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-provider-aws/internal/conns"
+)
+
+func ResourceDefaultVault() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceDefaultVaultCreate,
+		Read:   resourceDefaultVaultRead,
+		Delete: resourceDefaultVaultDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"arn": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"kms_key_arn": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"recovery_points": {
+				Type:     schema.TypeInt,
+				Computed: true,
+			},
+		},
+	}
+}
+
+func resourceDefaultVaultCreate(d *schema.ResourceData, meta interface{}) error {
+	conn := meta.(*conns.AWSClient).BackupConn
+
+	input := &backup.ListBackupVaultsInput{}
+	output, err := conn.ListBackupVaults(input)
+
+	if err == nil && len(output.BackupVaultList) > 0 {
+		vault := output.BackupVaultList[0]
+		log.Printf("[INFO] Found existing Backup Vault (%s)", *vault.BackupVaultName)
```

Comment:
```suggestion
		log.Printf("[INFO] Found existing Backup Vault (%s)", aws.StringValue(vault.BackupVaultName))
```

Для разыменования обычно используются утильные функции

Reply:
Поправил